forge-gui/res/adventure/common/decks/standard/pestilence.dck

```diff
@@ -0,0 +1,44 @@
+[duel]
+[metadata]
+Name=Micheal Myers's Blades of All Hallow's 3
+Title=Micheal Myers
+Difficulty=hard
```

Comment:
Some decks like this one seem to have Quest Mode-like metadata left in them, can probably be cleaned up, though shouldn't be critical as it likely just does nothing in Adventure Mode.